bin/configurator/static/_includes/en/configurator/partials/ci/gitlab_docker_main_section.md.liquid

```diff
@@ -1,79 +1,60 @@
 ### Requirements
 
 - GitLab;
-
-- Host to run GitLab Runner with:
+- Host for GitLab Runner, with:
 
   - [Docker Engine](https://docs.docker.com/engine/install/).
 
 ### Installing GitLab Runner
 
-Install GitLab Runner on its dedicated host by following the [official instructions](https://docs.gitlab.com/runner/install/linux-repository.html).
+Install GitLab Runner on the dedicated host using the [official instructions](https://docs.gitlab.com/runner/install/linux-repository.html).
 
 {% include configurator/partials/ci/buildah_ubuntu.md.liquid %}
 
 ### Registering GitLab Runner
 
-Follow [official instructions](https://docs.gitlab.com/runner/register/index.html) to register GitLab Runner in GitLab; specify Docker as the executor and any image as the image (e.g. `alpine`).
-
-### Configuring GitLab Runner
-
-On the GitLab Runner host, run the following commands:
+You can follow the [official registration guide](https://docs.gitlab.com/runner/register/index.html), or run the following command to register the Runner with Docker executor:
 
 ```bash
-# Create necessary directories and set permissions
-mkdir -p $HOME/.local/share/containers $HOME/.werf
-sudo chown -R 1000:1000 $HOME/.local/ $HOME/.werf
-chmod 755 $HOME/.local/share/containers
+gitlab-runner register \
```

Comment:
I don't like specific runner instructions here. What's $URL and $TOKEN? If you want to copy instructions from GitLab doc properly there would be a lot of copy-paste, and it will need constant sync.